layouts/partials/head.html

```diff
@@ -1,6 +1,6 @@
 <head>
   <meta charset="utf-8">
-  <title>{{ .Title }}</title>
+  <title> {{ .Title }} </title>
```

Comment:
We can avoid adding extra whitespace, since when this is rendered, it would be `<title> Site Title </title>`. Unexpected whitespace can be tricky at times, so best to avoid:
```suggestion
  <title>{{ .Title }}</title>
```